api/rpc/client/client.go

```diff
@@ -65,8 +74,12 @@ func NewPublicClient(ctx context.Context, addr string) (*Client, error) {
 }
 
 // NewClient creates a new Client with one connection per namespace with the
-// given token as the authorization token.
+// given token as the authorization token. In case if token will be empty, then
+// `CELESTIA_NODE_AUTH_TOKEN` will be used in order to get the token.
 func NewClient(ctx context.Context, addr string, token string) (*Client, error) {
+	if token == "" {
+		token = os.Getenv(authEnvKey)
+	}
```

Comment:
This is a security issue, we cannot do this.

Reply:
why?